source/reference/api/commit.txt

```diff
@@ -0,0 +1,102 @@
+.. _c2c-api-commit:
+
+==========
+``commit``
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Description
+-----------
+
+Commits the synchronization operation to the destination cluster.
+
+Requirement
+-----------
+
+To use the ``commit`` endpoint, the following conditions must be met:
+
+- The {+c2c-product-name+} must be in the ``RUNNING`` state. 
+- The ``canCommit`` boolean returned by the :ref:`progress
+  <c2c-api-progress>` endpoint must be ``true``.
+
+Request
+-------
+
+.. code-block:: http
+
+   POST /api/v1/commit 
+
+Request Body Parameters
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: includes/api/facts/no-body-parameters.rst
+
+Response
+--------
+
+.. include:: /includes/api/tables/basic-response.rst
+
+Example
+-------
+
+The following example commits the synchronization operation to the
+destination cluster.
+
+Confirm that the Synchronization is Ready
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before sending a request to the ``commit`` endpoint, use the
+:ref:`progress <c2c-api-progress>` endpoint to confirm that the
+synchronization is ready to be committed.
+
+Request
+```````
+
+.. literalinclude:: /includes/api/requests/progress.sh
+   :language: shell
+
+Response
+````````
+
+.. literalinclude:: /includes/api/responses/progress.json
+   :language: shell
+   :emphasize-lines: 5
+
+The ``progress`` endpoint returned ``"canCommit":true``, which means
+that the ``commit`` request can run successfully.
+
+Send the Commit Request
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The following command sends a request to the ``commit`` endpoint:
+
+Request
+```````
+
+.. literalinclude:: /includes/api/requests/commit.sh
+   :language: shell
+
+Response
+````````
+
+.. literalinclude:: /includes/api/responses/success.json
+   :language: json
+
+Behavior
+--------
+
+If the ``commit`` request is successful, {+c2c-product-name+} enters the
+``COMMITTING`` state, then automatically transitions to the
+``COMMITTED`` state.
+
+The amount of time that it takes {+c2c-product-name+} to transition from
+the ``COMMITTING`` state to the ``COMMITTED`` state is indicated by the
+``lagTimeSeconds`` in the :ref:`progress <c2c-api-progress>` endpoint
+output.
```

Comment:
we should highlight this point more prominently. See comment above

Reply:
Updated in the Requirements section. I think when we add a more guided tutorial we can go into even more detail. The goal of these pages is to largely be API reference.